src/common/Utilities/StringConvert.h

```diff
@@ -171,6 +170,91 @@ namespace Trinity::Impl::StringConvertImpl
             return (val ? "1" : "0");
         }
     };
+
+#if TRINITY_COMPILER == TRINITY_COMPILER_MICROSOFT
```

Comment:
@Shauren is there a better way to feature test from_chars support for `double`?

Reply:
using cmake compile or run test but i dont think bloating those is a good idea, this quick ifdef is fine